printing_auto_stock_picking/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_manager,stock_picking_auto_print - manager,printing_auto_base.model_printing_auto,stock.group_stock_manager,1,1,1,1
```

Comment:
Don't we need a record rule so that they can see auto-printing records only for stock models?

Reply:
Done. I had to add a model field to bound the config to it